ximea_ros2_cam/src/ximea_ros2_cam.cpp

```diff
@@ -91,7 +111,7 @@ XimeaROSCam::~XimeaROSCam() {
         RCLCPP_INFO_STREAM(this->get_logger(), "Closed device: " << this->cam_serialno_);
     }
 
-    RCLCPP_INFO(this->get_logger(), "ximea_ros2_cam node shutdown complete.");
+    RCLCPP_INFO(this->get_logger(), "%s node shutdown complete.", this->get_name());
```

Comment:
I create a macro called LOG_IF to be able to disable by parameter the debug mode (because the debugs was already in the package, and can be helpful considering the high number of camera configs). Commit: https://github.com/Brazilian-Institute-of-Robotics/ximea_ROS2_driver/commit/2bb6e0eae9117872aa357f1acf15b0cc064acba2